src/NuGetPackageVerifier/PackageIssueFactory.cs

```diff
@@ -8,6 +8,17 @@ namespace NuGetPackageVerifier
 {
     public static class PackageIssueFactory
     {
+        public static PackageVerifierIssue AssemblyMissingAuthorAttribute(string assemblyPath)
+        {
+            return new PackageVerifierIssue(
+                "AUTHOR_ATTRIBUTE",
+                assemblyPath,
+                string.Format(
+                    @"The managed assembly '{0}' in this package is missing the '[assembly: AssemblyMetadata(""Author"", ""<text>"")]' attribute.",
```

Comment:
Hmm I don't think that's where we get the author from. It seems that we get it from some other magical place: https://github.com/aspnet/KoreBuild/pull/93/files

cc @pranavkm @BrennanConroy